### DISCUSSION TITLE
WSL虚拟环境中如何将Alice和Bob分开？

### DISCUSSION THREAD

**Discussion by guuacel** [secretflow/secretflow#1285] [General]
WSL虚拟环境中如何将Alice和Bob分开？是将Alice和Bob分开在两个conda的虚拟环境中然后进行ray集群的创建吗？本人基础较差，我顺便想问一下，ip和端口号报错是为什么呢？报错信息如下：Global stats: 38 total (1 active)
Queueing time: mean = 42.237 us, max = 394.716 us, min = 20.656 us, total = 1.605 ms
Execution time:  mean = 4.230 us, total = 160.749 us
Event stats:
        GcsTaskManager.GcJobSummary - 37 total (1 active), Execution time: mean = 4.125 us, total = 152.615 us, Queueing time: mean = 32.711 us, max = 47.545 us, min = 20.656 us, total = 1.210 ms
        PeriodicalRunner.RunFnPeriodically - 1 total (0 active), Execution time: mean = 8.134 us, total = 8.134 us, Queueing time: mean = 394.716 us, max = 394.716 us, min = 394.716 us, total = 394.716 us

**Comment by wangzul** [comment]
两个conda的虚拟环境，ray、alice、bob 使用不同的端口即可。
ip和端口号报错，从描述来看信息比较少，可以提供双方节点的运行日志，方便的话建议代码也可以提供，可以加快解决效率
